futurex_openedx_extensions/dashboard/docs_src.py

```diff
@@ -1785,8 +1785,8 @@ def get_optional_parameter(path: str) -> Any:
 
     'ThemeConfigTenantView.post': {
         'summary': 'Create new tenant along with default theme config.',
-        'description': 'Create new tenant along with default theme config.\n'
-        '\n**Note:** This API is just mock API with dummy data and not implemented yet.',
+        'description': 'Create new tenant along with default theme config. The caller must have staff or fx '
+        'api global access.\n',
```

Comment:
No need for this to be mentioned since it's configurable through View Roles admin page
```suggestion
        'description': 'Create new tenant along with default theme config.\n',
```